underfs/local/src/test/java/alluxio/underfs/local/LocalUnderFileSystemTest.java

```diff
@@ -45,110 +45,6 @@ public void before() throws IOException {
   }
 
   @Test
-  public void exists() throws IOException {
```

Comment:
The tests in this file are verifying that `LocalUnderFileSystem` makes the right modifications to the actual filesystem, while the tests in `UnderStorageSystemInterfaceIntegrationTest` only verify that the UFS implementation is consistent with itself (all calls go through the UFS). If UFS refactors invalidate tests in `LocalUnderFileSystemTest`, we can delete them then, but for now I think it makes sense to have tests which verify by communicating directly with the local filesystem.